test/unit/test_pretty.py

```diff
@@ -0,0 +1,142 @@
+import json
+
+import jubilant
+
+from .fake_statuses import DATABASE_WEBAPP_JSON, MINIMAL_STATUS, SNAPPASS_JSON
+
+
+def test_small():
+    status_repr = repr(MINIMAL_STATUS)
+    assert (
+        status_repr
+        == "Status(model=ModelStatus(name='mdl', type='typ', controller='ctl', cloud='aws', version='3.0.0'), machines={}, apps={})"
+    )
+    assert str(MINIMAL_STATUS) == status_repr
+
+    # Ensure it's actually a Python repr: it can roundtrip via eval()
+    assert eval(status_repr, jubilant.statustypes.__dict__) == MINIMAL_STATUS
```

Comment:
This is an interesting approach.
I dunno if we want to give this guarantee to our users, but it sure is convenient :)

Reply:
Yeah, my intent is not to guarantee it to our users, but to make sure we've got it right ourselves. For our users, all the docstring says is "Return a pretty-printed version of the status".